### PR TITLE
load heated floor area data

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds functionality to transform and load extracted heated floor area data.

https://trello.com/c/q0fW8AUX/112-extract-load-heated-floor-data

### CODE REVIEW

**Comment by buckley-w-david** [comment]
[Extract/Load Heated Floor data](https://trello.com/c/q0fW8AUX/112-extract-load-heated-floor-data)

File: python/src/energuide/dwelling.py [hunk 1]
```diff
@@ -9,9 +9,11 @@
 from energuide.extracted_datatypes import Door
 from energuide.extracted_datatypes import Wall
 from energuide.extracted_datatypes import Window
+from energuide.extracted_datatypes import HeatedFloorArea
 from energuide.extracted_datatypes import Codes
 
 
+
```
Comment (JasonMWhite) [inline python/src/energuide/dwelling.py]: 🔥

File: python/src/energuide/extracted_datatypes.py [hunk 2]
```diff
@@ -193,6 +198,44 @@ def from_data(cls, codes: typing.Dict[str, typing.List[typing.Dict[str, str]]]):
             window=window_codes,
         )
 
+class HeatedFloorArea(_HeatedFloorArea):
+
+
```
Comment (JasonMWhite) [inline python/src/energuide/extracted_datatypes.py]: 🔥  1 extra newline

File: python/tests/test_dwelling.py [hunk 3]
```diff
@@ -299,6 +311,12 @@ def test_from_row(self, sample_input_d: reader.InputData) -> None:
                     height=1.3220699,
                 )
             ],
+            heated_floor_area=[
+                extracted_datatypes.HeatedFloorArea(
+                    area_above_grade=185.8,
+                    area_below_grade=92.9,
+                )
+            ]
```
Comment (JasonMWhite) [inline python/tests/test_dwelling.py]: end in `,`

**Pull request merged**